WalletWasabi.Daemon/Config.cs

```diff
@@ -21,36 +22,88 @@ public Config(PersistentConfig persistentConfig, string[] cliArgs)
-			{ nameof(JsonRpcServerEnabled), GetBoolValue("JsonRpcServerEnabled", PersistentConfig.JsonRpcServerEnabled, cliArgs) },
-			{ nameof(JsonRpcUser), GetStringValue("JsonRpcUser", PersistentConfig.JsonRpcUser, cliArgs) },
-			{ nameof(JsonRpcPassword), GetStringValue("JsonRpcPassword", PersistentConfig.JsonRpcPassword, cliArgs) },
-			{ nameof(JsonRpcServerPrefixes), GetStringArrayValue("JsonRpcServerPrefixes", PersistentConfig.JsonRpcServerPrefixes, cliArgs) },
-			{ nameof(RpcOnionEnabled), GetBoolValue("RpcOnionEnabled", value: false, cliArgs) },
-			{ nameof(DustThreshold), GetMoneyValue("DustThreshold", PersistentConfig.DustThreshold, cliArgs) },
-			{ nameof(BlockOnlyMode), GetBoolValue("BlockOnly", value: false, cliArgs) },
-			{ nameof(LogLevel), GetStringValue("LogLevel", value: "", cliArgs) },
-			{ nameof(EnableGpu), GetBoolValue("EnableGpu", PersistentConfig.EnableGpu, cliArgs) },
-			{ nameof(CoordinatorIdentifier), GetStringValue("CoordinatorIdentifier", PersistentConfig.CoordinatorIdentifier, cliArgs) },
+			[ nameof(Network)] = (
+				"The Bitcoin network to use: main, testnet, or regtest",
+				GetNetworkValue("Network", PersistentConfig.Network.ToString(), cliArgs)),
+			[ nameof(MainNetBackendUri)] = (
+				"The backend server's URL to connect to when the Bitcoin network is main",
+				GetStringValue("MainNetBackendUri", PersistentConfig.MainNetBackendUri, cliArgs)),
+			[ nameof(TestNetBackendUri)] = (
+				"The backend server's URL to connect to when the Bitcoin network is testnet",
+				GetStringValue("TestNetBackendUri", PersistentConfig.TestNetBackendUri, cliArgs)),
+			[ nameof(RegTestBackendUri)] = (
+				"The backend server's URL to connect to when the Bitcoin network is regtest",
+				GetStringValue("RegTestBackendUri", PersistentConfig.RegTestBackendUri, cliArgs)),
+			[ nameof(MainNetCoordinatorUri)] = (
+				"The coordinator server's URL to connect to when the Bitcoin network is main",
+				GetNullableStringValue("MainNetCoordinatorUri", PersistentConfig.MainNetCoordinatorUri, cliArgs)),
+			[ nameof(TestNetCoordinatorUri)] = (
+				"The coordinator server's URL to connect to when the Bitcoin network is testnet",
+				GetNullableStringValue("TestNetCoordinatorUri", PersistentConfig.TestNetCoordinatorUri, cliArgs)),
+			[ nameof(RegTestCoordinatorUri)] = (
+				"The coordinator server's URL to connect to when the Bitcoin network is regtest",
+				GetNullableStringValue("RegTestCoordinatorUri", PersistentConfig.RegTestCoordinatorUri, cliArgs)),
+			[ nameof(UseTor)] = (
+				"All the communications go through the Tor network",
+				GetBoolValue("UseTor", PersistentConfig.UseTor, cliArgs)),
+			[ nameof(TerminateTorOnExit)] = (
+				"Stop the Tor process when Wasabi is closed",
+				GetBoolValue("TerminateTorOnExit", PersistentConfig.TerminateTorOnExit, cliArgs)),
+			[ nameof(DownloadNewVersion)] = (
+				"Automatically download any new released version of Wasabi",
```

Comment:
Is it any new version, or just the major version?

Reply:
any new version.
In the GUI the _update available_ message is only with major version.
Auto download always gets the latest release